src/gausskernel/storage/access/k2/pg_gate_api.cpp

```diff
@@ -100,6 +101,7 @@ namespace {
 void PgGate_InitPgGate(const K2PgTypeEntity *k2PgDataTypeTable, int count, PgCallbacks pg_callbacks) {
     assert(pg_gate == nullptr && "PgGate should only be initialized once");
     elog(INFO, "K2 PgGate open");
+    K2LOG_I(::k2pg::log::k2pg, "Initializing Pggate");
```

Comment:
try to use elog only here. don't need K2LOG_I here since we have elog already.

Reply:
Removed.